Assets/!/Scripts/PlayerAudio.cs

```diff
@@ -1,57 +1,100 @@
-using System.Diagnostics;
+using System.Collections.Generic;
 using UnityEngine;
 using utils;
 
 public class PlayerAudio : MonoBehaviour
 {
-    public AudioSource audioSource;
-    public Audio[] audioPlayer;
+    public AudioSource AudioSource;
+
+    public Audio[] WalkAudio;
+    public Audio[] RunAudio;
+    public Audio[] DieScreamAudio;
+    public Audio[] BloodAudio;
+    public Audio[] LeftCrouchAudio;
+    public Audio[] RightCrouchAudio;
+    public Audio[] CrawlAudio;
+    public Audio[] BulletHitAudio;
+    public Audio[] JumpAudio;
+    public Audio[] FallDownAudio;
 
     void Start()
     {
-        audioSource = GetComponent<AudioSource>();
+        AudioSource = GetComponent<AudioSource>();
     }
 
     public void Walk()
     {
-        PlayAudio("Walk", 0.8f);
+        PlayRandomAudio(WalkAudio, 0.8f);
+    }
+
+    public void LeftCrouched()
+    {
+        PlayRandomAudio(LeftCrouchAudio, 0.5f);
     }
 
-    public void Crouched()
+    public void RightCrouched()
     {
-        PlayAudio("Crouched", 0.5f);
+        PlayRandomAudio(RightCrouchAudio, 0.5f);
     }
 
     public void Crawl()
     {
-        PlayAudio("Crawl", 0.20f);
+        PlayRandomAudio(CrawlAudio, 0.20f);
     }
 
     public void Run()
     {
-        PlayAudio("Walk", 1f);
+        PlayRandomAudio(RunAudio, 1f);
+    }
+
+    public void DieScream()
+    {
+        PlayRandomAudio(DieScreamAudio, 0.8f);
+    }
+
+    public void Blood()
+    {
+        PlayRandomAudio(BloodAudio, 3.5f);
+    }
+
+    public void BulletHit()
+    {
+        PlayRandomAudio(BulletHitAudio, 1.5f);
     }
 
     public void Jump()
     {
-        PlayAudio("Jump", 1f);
+        PlayRandomAudio(JumpAudio, 1f);
     }
 
-    public void PlayAudio(string clipName, float volume)
+    public void FallDown()
     {
-        Audio audioToPlay = System.Array.Find(audioPlayer, a => a.id == clipName);
+        PlayRandomAudio(FallDownAudio, 0.7f);
+    }
 
-        if (audioToPlay.clip == null)
+    private void PlayRandomAudio(Audio[] audioArray, float volume)
+    {
+        if (audioArray.Length == 0)
         {
-            UnityEngine.Debug.LogWarning("Audio clip not found: " + clipName);
+            Debug.LogWarning("Audio array is empty");
             return;
         }
 
-
-        audioSource.clip = audioToPlay.clip;
-        audioSource.volume = volume;
-        audioSource.Play();
-        audioSource.loop = false;
+        Audio audioClip = audioArray.Length == 1 ? audioArray[0] : audioArray[Random.Range(0, audioArray.Length)];
```

Comment:
No need for the ternal: `Random.Range(0, audioArray.Length)` should return 0 if length is 1